src/System.Private.ServiceModel/tests/Scenarios/Security/TransportSecurity/Https/HttpsTests.cs

```diff
@@ -161,7 +161,9 @@ public static void SameBinding_Soap12_EchoString()
     }
 
     [Fact]
-    [ActiveIssue(544, PlatformID.AnyUnix)]
+#if FEATURE_NETNATIVE
+    [ActiveIssue(544)] // Server certificate validation not supported in NET Native
+#endif
```

Comment:
Not sure, @mconnew would know. The issue implies that it works on Windows but not Unix, there is a later comment that says it is now supported on UWP.
If Matt fixed it and it now works on Unix he should verify it and remove the ActiveIssue?